src/Stratis.Bitcoin.Features.BlockStore/AddressIndexing/AddressIndexer.cs

```diff
@@ -70,6 +70,8 @@ public class AddressIndexer : IAddressIndexer
 
         private const string AddressIndexerDatabaseFilename = "addressindex.litedb";
 
+        private const int FallBackMaxReorg = 200;
```

Comment:
Can you add a comment here to explain what the fall back is and how its set?

Reply:
I'd actually just set max reorg to BTC in the network class